pkg/client/observatorium/observability_config.go

```diff
@@ -118,11 +120,31 @@ func (c *ObservabilityCloudWatchLoggingConfig) setDefaults() {
 	}
 }
 
+func (c *ObservabilityCloudWatchLoggingConfig) GetEnterpriseCredentials(orgID string) map[string]string {
+	idx, enterpriseCredential := arrays.FindFirst(c.EnterpriseCredentials, func(enterpriseCredential ObservabilityEnterpriseCloudwatchLoggingConfigCredentials) bool {
+		return orgID == enterpriseCredential.OrgID
+	})
+	if idx != arrays.ElementNotFound {
+		stringDataMap := map[string]string{
```

Comment:
Can we return a `ObservabilityCloudwatchLoggingConfigCredentials` instead of a map of string,string?
That way we make this method independent on K8s secrets details

Reply:
:+1: